bin/alluxio

```diff
@@ -11,7 +11,11 @@
 #
 
 function printUsage {
-  echo "Usage: alluxio COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo "Usage: alluxio [-jvmdebug] COMMAND [GENERIC_COMMAND_OPTIONS] [COMMAND_ARGS]"
+  echo
+  echo "Flag description:"
+  echo -e "  -jvmdebug     \t If this flag specified, the env variable ALLUXIO_USER_DEBUG_JAVA_OPTS will be added to jvm flags."
```

Comment:
@aaudiber , Sorry, i gave the wrong command case, i really want to say is `format`, you can try it, and find that the `-debug` argument let the `alluxio` script think the $# is 1, but format command assert $1 must be `-s`, if not, output the usage. If move the -debug flag before the generic COMMAND, i can shift the -debug command in the early stage, the generic command  no need to make any modification.